src/snapshot/index.js

```diff
@@ -34,7 +34,8 @@ var Snapshot = {
     clone: require('./cloneplot'),
     toSVG: require('./tosvg'),
     svgToImg: require('./svgtoimg'),
-    toImage: require('./toimage')
+    toImage: require('./toimage'),
+    toSnapshot: require('./tosnapshot')
```

Comment:
Oops. Scratch  :arrow_double_up: , `toSnapshot` is essentially what the modebar uses. Putting that in a separate file in `src/snapshot/` makes sense. 

Maybe you could rename `tosnapshot.js` -> `download.js` instead?